### PR TITLE
Add tour model

### PR DESCRIPTION
**Pull request by mariorzgz**
Added tour model
Added associations between tour and user
Added validations for tour model and user model

### CODE REVIEW

File: app/models/tour.rb [hunk 1]
```diff
@@ -1,3 +1,10 @@
 class Tour < ApplicationRecord
   belongs_to :user
+
+  validates :name, presence: true
+  validates :city, presence: true
+  validates :starting_point, presence: true
+  validates :description, presence: true, length: { minimum: 100 }
```
Comment (yigittuncel) [inline app/models/tour.rb]: can you also add a maximum number of characters, please?

File: app/models/user.rb [hunk 2]
```diff
@@ -3,4 +3,11 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
```
Comment (yigittuncel) [inline app/models/user.rb:6]: Don't we need has_many :bookings as, well?

**Review by yigittuncel — CHANGES_REQUESTED** [review]

File: app/models/user.rb (hunk 2 above)
Comment (mariorzgz) [reply app/models/user.rb:6]: Yes, but we don't have a booking model yet, it's better to add it later

File: app/models/tour.rb (hunk 1 above)
Comment (mariorzgz) [reply app/models/tour.rb]: Does 500 maximum sound good? Maybe some people want to add information about the tour or maybe it's a really long tour

File: app/models/user.rb (hunk 2 above)
Comment (yigittuncel) [reply app/models/user.rb:6]: that's true 👍

File: app/models/tour.rb (hunk 1 above)
Comment (mariorzgz) [reply app/models/tour.rb]: Done!

**Review by yigittuncel — APPROVED** [review]

**Pull request merged**